manifests/g/GIMP/GIMP/Nightly/2.99.4/GIMP.GIMP.Nightly.installer.yaml

```
@@ -1,17 +1,29 @@
-# Created with YamlCreate.ps1 v2.1.2 $debug=AUSU.5-1-19041-1320
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.7-2-5
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
 
 PackageIdentifier: GIMP.GIMP.Nightly
 PackageVersion: 2.99.4
+Platform:
+- Windows.Desktop
 MinimumOSVersion: 10.0.0.0
 InstallerType: inno
-InstallerSwitches:
-  Silent: /VERYSILENT
-  SilentWithProgress: /SILENT
-  Custom: /NORESTART /ALLUSERS
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
 Installers:
-- Architecture: x64
-  InstallerUrl: https://download.gimp.org/pub/gimp/v2.99/windows/gimp-2.99.4-setup.exe
-  InstallerSha256: 3ECD4E88972E8F4D38F70D73B6236C3433C9EA2A60DFD0C9E2395B83280092ED
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://download.gimp.org/pub/gimp/v2.99/windows/gimp-2.99.4-setup-1.exe
+  InstallerSha256: 243BD00741DE1E25A91EC1AE5AF13D9A1ACF6DACD1486840274E1DCE0A79AB90
+  InstallerSwitches:
+    Custom: /NORESTART /CURRENTUSER
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://download.gimp.org/pub/gimp/v2.99/windows/gimp-2.99.4-setup-1.exe
+  InstallerSha256: 243BD00741DE1E25A91EC1AE5AF13D9A1ACF6DACD1486840274E1DCE0A79AB90
+  InstallerSwitches:
+    Custom: /NORESTART /ALLUSERS
```

This should be supported on both x86 and x64 systems
```suggestion
- Architecture: x86
  Scope: user
  InstallerUrl: https://download.gimp.org/pub/gimp/v2.99/windows/gimp-2.99.4-setup-1.exe
  InstallerSha256: 243BD00741DE1E25A91EC1AE5AF13D9A1ACF6DACD1486840274E1DCE0A79AB90
  InstallerSwitches:
    Custom: /NORESTART /CURRENTUSER
- Architecture: x86
  Scope: machine
  InstallerUrl: https://download.gimp.org/pub/gimp/v2.99/windows/gimp-2.99.4-setup-1.exe
  InstallerSha256: 243BD00741DE1E25A91EC1AE5AF13D9A1ACF6DACD1486840274E1DCE0A79AB90
  InstallerSwitches:
    Custom: /NORESTART /ALLUSERS
- Architecture: x64
  Scope: user
  InstallerUrl: https://download.gimp.org/pub/gimp/v2.99/windows/gimp-2.99.4-setup-1.exe
  InstallerSha256: 243BD00741DE1E25A91EC1AE5AF13D9A1ACF6DACD1486840274E1DCE0A79AB90
  InstallerSwitches:
    Custom: /NORESTART /CURRENTUSER
- Architecture: x64
  Scope: machine
  InstallerUrl: https://download.gimp.org/pub/gimp/v2.99/windows/gimp-2.99.4-setup-1.exe
  InstallerSha256: 243BD00741DE1E25A91EC1AE5AF13D9A1ACF6DACD1486840274E1DCE0A79AB90
  InstallerSwitches:
    Custom: /NORESTART /ALLUSERS
```